src/data-import/canvas-file.ts

```diff
@@ -0,0 +1,25 @@
+/** importer for canvas nodes */
+
+// import { CanvasMetadata } from "data-model/canvas";
+import { CanvasCard, CanvasMetadataIndex } from "data-model/canvas";
+import { FileStats /* parseYaml */ } from "obsidian";
+import { parsePage } from "./markdown-file";
+
+export function parseCanvasCard(
+    path: string,
+    id: string,
+    contents: string,
+    stat: FileStats,
+    mindex: CanvasMetadataIndex
+) {
+    // @ts-expect-error SHUT UP MEG
+    const metadata = mindex[path]?.caches[id];
```

Comment:
the error is:
> Element implicitly has an 'any' type because expression of type 'string' can't be used to index type 'CachedMetadata'.
>  No index signature with a parameter of type 'string' was found on type 'CachedMetadata'.ts(7053)